.travis.yml

```diff
@@ -17,8 +17,16 @@ before_install:
   - pecl install -f mongodb
   - composer validate --strict
 
-before_script:
-  - composer install
+install:
+  - composer update --prefer-stable --prefer-dist ${COMPOSER_FLAGS}
 
 script:
   - ./vendor/bin/phpunit -v
+
+jobs:
+  include:
+    # Tests the lowest set of dependencies
+    - php: 7.2
+      env: LOWEST COMPOSER_FLAGS="--prefer-lowest"
```

Comment:
Is this in case we remove the `install` override below in the future?

Reply:
Yep, forgot to remove that. It's gone now.